tui/methods.sh

```diff
@@ -23,6 +23,9 @@ if [ "$EXISTING_INSTANCE" == "true" ]; then
   available_methods=("$INSTANCE_TYPE")
 fi
 
+# adding back button
+available_methods+=($BACK_BUTTON)
```

Comment:
Same as above, `$BACK_BUTTON` should not be in this array and I think it should not be in any array at all.